ci/ccp_ci_unittests.sh

```diff
@@ -49,29 +49,36 @@ echo "=========================================================="
 sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
 ssh_cmd="ssh $sshopts $CICO_hostname"
 
+echo "====Installing necessary packages for running tests on the duffy node===="
 # install the needed packages on the duffy node which will the tests
-$ssh_cmd "yum -y install epel-release && yum -y install rsync git PyYAML python-networkx python2-nose"
+$ssh_cmd "yum -y install epel-release && yum -y install rsync git PyYAML python-networkx python2-nose" >> /tmp/unittests_setup_log.txt
 package_installed_success=$?
 
-if [ $package_installed_success -ne 0]
+if [ $package_installed_success -ne 0 ]
 then
-
+  echo "==================Package Installation Logs============"
+  cat /tmp/unittests_setup_log.txt
```

Comment:
We `echo` package installation logs but the file is named as `unittests_setup_log.txt`. Isn't that awkward?

Reply:
this is package installation log for setting up unittests. so the echo is `package installation logs` because the job name itself is unit test.
 We can change the file name to anything, but this will not be visible to any one anyway.

Reply:
Yep, I understand the visibility point. It was only from dev perspective.